core/templates/components/skill-selector/skill-selector.component.html

```diff
@@ -0,0 +1,105 @@
+<div class="skill-selector-parent skill-container oppia-skill-container protractor-test-skill-container">
+  <div class="list-group oppia-list-group">
+    <mat-card layout="row" class="list-view-item protractor-test-skill-select-header">
+      Search By Skill
+      <input type="text" class="form-control protractor-test-skill-name-input" placeholder="Enter a skill name" ng-model="skillFilterText" autofocus>
+    </mat-card>
+    <div class="skill-list-box">
+      <mat-radio-group ng-model="selectedSkill" ng-change="setSelectedSkillId()">
```

Comment:
This would change to [(ngModel)] and (change) respectively.

Reply:
Done